blocks/core/ff_module/ff_module-profile-response-button/_ff_module-profile-response-button-renderer.js

```diff
@@ -10,9 +10,11 @@ module.exports = function() {
                 onSelect: function() {
                     console.log("onSelect");
                 },
-                isSelected: false,
+                uiState: "",
                 guid: "u42",
                 label: "Sally Student",
+                status: "Resubmission Requested",
+                mark: "B, 76%",
```

Comment:
In a branch, I have the mark being passed in a rich object i.e.

```
markAndGrade: {
    mark: 7,
    markMax: 10,
    grade: "A"
}
```

where some of the properties may be missing. That leaves the decision of how to actually format the mark and grade to the component, which seems to me like the right place to decide.